db/migrate/20210824161014_add_bio_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddBioToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :bio, :text
```

Comment:
Unclear what text means as there is no text column, in the database. Is it referencing username?

Reply:
text refers to the data that is being put into the bio